packages/components/src/lib/controllers/ui-controller.js

```diff
@@ -284,7 +274,13 @@ if you want to create a different configuration of a UI controller.
       triggerPreCallback: null,
       // if true, the help button on the toolbar can be controlled by the client, no click handler will
       // be added by the components library
-      overrideHelp: false
+      overrideHelp: false,
+      /*
+      This is an experimental flag that will force treebank iframe URL to be reset every time a treebank tab is
+      opened. It will force a content of an iframe to be reloaded and thus will solve display issues
+      with a treebank view.
+       */
+      experimentalResetTreebankURL: false
```

Comment:
I have made the default false because it is working well with the Arethusa fix in place.